### PR TITLE
WIP, has to be tested: upgrade to sbt 1.1.1

### PR DESCRIPTION
**Pull request by fabianpage**

### CODE REVIEW

**Comment by fabianpage** [comment]
The build.sbt needs also some cleanup

**Review by AlwinEgger — APPROVED** [review]
Checked with data from initiative, seems to work fine.

File: build.sbt [hunk 1]
```diff
@@ -5,12 +5,6 @@ enablePlugins(JavaServerAppPackaging)
 name := "openolitor-server"
 mainClass in Compile := Some("ch.openolitor.core.Boot")
 
-sources in EditSource <++= baseDirectory.map(d => (d / "manifest_test.yml").get)
-sources in EditSource <++= baseDirectory.map(d => (d / "manifest_int.yml").get)
-sources in EditSource <++= baseDirectory.map(d => (d / "manifest_prod.yml").get)
-variables in EditSource += ("foo", "bar")
-targetDirectory in EditSource <<= baseDirectory(_ / "target")
-
```
Comment (AlwinEgger) [inline build.sbt:13]: Check needed if manifests are still in build package!

Comment (AlwinEgger) [reply build.sbt:13]: I checked. It's not as the manifest-files do not exist any more.

**Pull request merged**